Update index.md (#4467)
alphagov/govuk-design-system: src/community/upcoming-components-patterns/index.md

To get a wider look at our work and see what we released recently, see our [Roadmap](/community/roadmap/).

## Essential component criteria

> - Our way of prioritising work, alongside community input
> - List of essential criteria

Reviewer: ```suggestion
Alongside community input, we prioritise work to maintain and improve components that are essential to the Design System.
```

>   
> 1. Build things for user facing services on GOV.UK.
> 2. Required to provide security and trust in GOV.UK through consistent design across its services
> 3. Provides the building blocks to allow teams to successfully iterate at scale
> 4. Improves accessibility of GOV.UK services
> 5. Improves sustainability of GOV.UK services
> 6. Used in a large number of services
> 

Reviewer: ```suggestion  
An essential component should:

1. build things for user facing services on GOV.UK.
2. be required to provide security and trust in GOV.UK through consistent design across its services
3. provide the building blocks to allow teams to successfully iterate at scale
4. improve accessibility of GOV.UK services
5. improve sustainability of GOV.UK services
6. be in use in a large number of services
```

        ]
      }

>     }
>     {
>       key: {
>         text: "Essential components"
>       },
>       value: {
>         text: "We're working to improve our essential components"
>       },
>       
>   ]

Reviewer 2: ```suggestion
    },
    {
      key: {
        text: "Essential components"
      },
      value: {
        text: "We're working to improve our essential components"
      }
    }
  ]
```

Had a comma in the wrong place and an unclosed bracket.
